SolARService_MappingAndRelocFrontend/tests/SolARServiceTest_MappingAndRelocFrontend/SolARServiceTest_MappingAndRelocFrontend.cpp

```diff
@@ -201,6 +217,45 @@ int main(int argc, char* argv[])
         auto overlay3D = componentMgr->resolve<display::I3DOverlay>();
         LOG_INFO("Remote producer client: AR device component created");
 
+#ifdef IS_GT_SEQ
+        // load GT pose data
+        std::string pathToData = arDevice->bindTo<xpcf::IConfigurable>()->getProperty("pathToData")->getStringValue();
+        std::ifstream infile(pathToData + "/solar_to_world.txt");
+        std::string line;
+        if (!infile.is_open()) {
+            g_solar2world = Transform3Df::Identity();
+        }
+        else {
+            for (int r = 0; r < 4; r++) {
```

Comment:
Could be simpler if the matrix was stored on one line ? (in scenarios where we want to store multiple matrices, it makes more sense).
Maybe we need to factorize these kind of stuff somewhere. It's linked to the comment below, so keep as is for now, we need to address this properly, when a work on testing will be started.